<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Conversor</class>
 <widget class="QWidget" name="Conversor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conversor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/Recursos/Imagenes/icon_temperatura.png</normaloff>:/Recursos/Imagenes/icon_temperatura.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="font">
         <font>
          <family>Terminal</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
color: rgb(113, 96, 212);
background-color: rgb(63, 63, 63);
</string>
        </property>
        <property name="title">
         <string>Centigrados</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLCDNumber" name="outCent">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(85, 87, 83);
color: rgb(78, 154, 6);
</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="inCent">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="minimum">
            <number>-18</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox2">
        <property name="font">
         <font>
          <family>Terminal</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(55, 64, 235);

background-color: rgb(63, 63, 63);
</string>
        </property>
        <property name="title">
         <string>Fahrenheit</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLCDNumber" name="outFahr">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(85, 87, 83);
color: rgb(143, 89, 2);</string>
           </property>
           <property name="intValue" stdset="0">
            <number>32</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="intFahr">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>210</number>
           </property>
           <property name="value">
            <number>32</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox3">
        <property name="font">
         <font>
          <family>Terminal</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(104, 207, 60);

background-color: rgb(63, 63, 63);
</string>
        </property>
        <property name="title">
         <string>Kelvin</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLCDNumber" name="outKelv">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(85, 87, 83);
color: rgb(36, 124, 117);</string>
           </property>
           <property name="intValue" stdset="0">
            <number>273</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="intKelv">
           <property name="styleSheet">
            <string notr="true">color: rgb(63, 65, 71);</string>
           </property>
           <property name="minimum">
            <number>255</number>
           </property>
           <property name="maximum">
            <number>372</number>
           </property>
           <property name="value">
            <number>273</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;a href=&quot;https://www.flaticon.es/iconos-gratis/temperatura&quot; title=&quot;temperatura iconos&quot;&gt;Temperatura iconos creados por Freepik - Flaticon&lt;/a&gt; </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>intFahr</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outFahr</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>intKelv</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outKelv</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inCent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outCent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
